### PR TITLE
more changes

### PR DESCRIPTION
**Pull request by matthauck**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/matthauck/review-test/pull/5" target="_blank">ReviewNinja

**Comment by matthauck** [comment]
a comment form github

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # review-test
 
 Add some text. and then some more.
+last one.
```
Comment (matthauck) [inline README.md:4]: nice one!

Comment (matthauck) [inline README.md:4]: thanks!

**Pull request closed without merging**